documentation/modules/rn-27-resolved-issues.adoc

```diff
@@ -7,6 +7,11 @@
 
 {project-first} 2.7 has the following resolved issues:
 
+[id="resolved-issues-2-7-1_{context}"]
+== Resolved issues 2.7.1
+
+
```

Comment:
```suggestion
.Disk transfer tasks are not completed although the migration is successful

In earlier releases of {project-short}, when migrating vSphere or OVA virtual machines (VMs), the disk transfer phase was completed successfully, but the individual tasks were not. This happened because the logs from virt-v2v did not fully scan and therefore missed the last indication that the copy phase was done. link:https://issues.redhat.com/browse/MTV-1310[(MTV-1310)]
```